plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -61,26 +62,41 @@ public void createControl(Composite parent) {
         AnalyticsEvents.APP_ENGINE_NEW_PROJECT_WIZARD_TYPE_NATIVE, parent.getShell());
 
     Composite container = (Composite) getControl();
+    GridLayoutFactory.fillDefaults().numColumns(1).extendedMargins(0, 0, 0, 8).applyTo(container);
```

Comment:
I think it's better to get that branch out of this PR. One way I know is (there may be a better way)

```
git reset 1118 (the branch you based off of)
git add
git commit
git push -f
```